pkg/3scale/amp/component/system_postgresql.go

```diff
@@ -128,8 +128,8 @@ func (p *SystemPostgreSQL) Deployment(containerImage string) *k8sappsv1.Deployme
 								},
 							},
 							Env: []v1.EnvVar{
-								helper.EnvVarFromSecret("POSTGRESQL_USER", SystemSecretSystemDatabaseSecretName, SystemSecretSystemDatabaseUserFieldName),
-								helper.EnvVarFromSecret("POSTGRESQL_PASSWORD", SystemSecretSystemDatabaseSecretName, SystemSecretSystemDatabasePasswordFieldName),
+								helper.EnvVarFromValue("POSTGRESQL_USER", p.Options.User),
+								helper.EnvVarFromValue("POSTGRESQL_PASSWORD", p.Options.Password),
 								// TODO This should be gathered from secrets but we cannot set them because the URL field of the system-database secret
 								// is already formed from this contents and we would have duplicate information. Once OpenShift templates
 								// are deprecated we should be able to change this.
```

Comment:
Same as above - Should we remove this comment since we are now solely parsing these vars from values instead of secrets? Or should these be gathered from secrets?

CC @MStokluska